apps/video/js/thumbnail_list.js

```diff
@@ -47,7 +47,8 @@ function ThumbnailList(groupClass, container) {
 
   // If the locale changes while we're running, then we need to
   // re-localize the dates in the group headers
-  navigator.mozL10n.ready(this.localize.bind(this));
+  this.localize();
+  document.addEventListener('DOMRetranslated', this.localize.bind(this));
```

Comment:
It does. The old code fired the callback when l10n was ready, and then when DOM was retranslated.
We could do:
```
document.l10n.ready.then(this.localize.bind(this));
document.addEventListener('DOMRetranslated', this.localize.bind(this));
```

but since everything that `this.localize` uses is asynchronous, we don't have to. :)